### PR TITLE
Piece wise readerwriter units

### PR DESCRIPTION
**Pull request by JoshuaSBrown**
Added default units to all the file readers and writers, as well as default units for csg.

### CODE REVIEW

**Review by JensWehner — DISMISSED** [review]
Can you explain a bit more what this does. please?

**Comment by JoshuaSBrown** [comment]
@JensWehner this pull request does very little at the moment. I have added the default units of each of the file readers and writers, in coming pull requests I will be using the default units given in this pull request to convert between them using the unitconverter class in tools. That way you can read a pdb file in its default units and automatically convert it to a gromacs file in its default units.

File: include/votca/csg/pdbwriter.h [hunk 1]
```diff
@@ -22,12 +22,15 @@
...
 #include <votca/csg/trajectorywriter.h>
 #include <votca/tools/constants.h>
+#include <votca/tools/unitconverter.h>
 
 namespace votca {
 namespace csg {
 
 class PDBWriter : public TrajectoryWriter {
  public:
+  const tools::DistanceUnit distance_unit = tools::DistanceUnit::angstroms;
```
Comment (junghans) [inline include/votca/csg/pdbwriter.h:32]: Why `tools::` and not `csg::`?

**Review by junghans — APPROVED** [review]

File: include/votca/csg/pdbwriter.h (hunk 1 above)
Comment (JoshuaSBrown) [reply include/votca/csg/pdbwriter.h:32]: All of the types come from the unitconverter header in tools.

**Pull request merged**